### PR TITLE
Improve metafiles

### PR DESCRIPTION
**Pull request by sdirix**
Update `.gitignore`, `package.json` and `README.md`

### CODE REVIEW

**Comment by eneufeld** [comment]
@edgarmueller @sdirix  should we add this [https://github.com/istanbuljs/nyc#include-reports-for-files-that-are-not-required](url) to the report, to make the coverage realistic?

**Comment by eneufeld** [comment]
we probably should configure nyc to check only the src folder see [https://github.com/istanbuljs/nyc#include-reports-for-files-that-are-not-required](https://github.com/istanbuljs/nyc#include-reports-for-files-that-are-not-required)

File: README.md [hunk 1]
```diff
@@ -1,39 +1,60 @@
...
-### Complex AngularJS Forms in the blink of an eye
+### Complex Forms in the blink of an eye
 
 JSONForms uses HTML custom elements and eliminates the need to write HTML templates in order to
 create forms by leveraging the capabilities of JSON and JSON schema.
 
 # Usage
-1. Install JSON Forms via `npm install jsonforms`
-2. Load `jsonforms-example.css` in the head section:
+To use JSONForms add `native-shim.js` and `jsonforms.js` to your HTML in this order. Once you add a `json-forms` element to the DOM with at least a `data` attribute, a form will be rendered for you. Data and UI schemas can be configured by the `dataSchema` and `uiSchema` attributes. Use CSS to style the form however you want.
```
Comment (edgarmueller) [inline README.md]: Update wording: ..with at least a `data` attribute [set]

File: README.md [hunk 2]
```diff
@@ -1,39 +1,60 @@
...
+      var jsonForms = document.createElement('json-forms');
+      jsonForms.data = {name:'John Doe'};
+      jsonForms.dataSchema = {type: "object", properties: {name : { type: "string"}}};
+      jsonForms.uiSchema = {type: "Control", scope: { $ref: "#/properties/name" } };
+      document.body.appendChild(jsonForms);
+    </script>
+  ```
+
 # Documentation and more information
 For documentation, examples and more information, please see [jsonforms.org](http://github.eclipsesource.com/jsonforms/).
```
Comment (edgarmueller) [inline README.md]: I'd remove for now as the link to the website hasn't been updated yet.

File: README.md [hunk 3]
```diff
@@ -60,12 +81,12 @@ Current status: [![Build Status](https://travis-ci.org/eclipsesource/jsonforms.s
  * Run either ```npm run publish-patch```,```npm run publish-minor``` or ```npm run publish-major```.
```
Comment (edgarmueller) [inline README.md]: Add header titled 'Developer documentation' or something the like

**Comment by edgarmueller** [comment]
Thanks

**Pull request merged**